### ISSUE TITLE
Include padding_idx parameter for EmbeddingBag

### ISSUE THREAD

**Issue by uwaisiqbal** [pytorch/pytorch#47858]
## 🚀 Feature

With `nn.Embedding` I can clearly specify the `padding_idx` so the model correctly understands how to deal with padding tokens. However, when I am using `nn.EmbeddingBag` I cannot specify the `padding_idx` parameter so the model can be thrown by the average being skewed with longer sequences of text with lots of padding. 

This feature would be very useful when dealing with variable length sequences so that the padding tokens can be dealt with properly by the model.

I would expect the behaviour to be consistent across the two modules but that is not the case. 

The request is to update the implementation for `nn.EmbeddingBag` so that it supports the `padding_idx` argument.

**Comment by ezyang** [comment]
dupe of https://github.com/pytorch/pytorch/issues/3194
